WalletWasabi.Daemon/Config.cs

```diff
@@ -191,9 +191,12 @@ [ nameof(AbsoluteMinInputCount)] = (
 	public bool EnableGpu => GetEffectiveValue<BoolValue, bool>(nameof(EnableGpu));
 	public string CoordinatorIdentifier => GetEffectiveValue<StringValue, string>(nameof(CoordinatorIdentifier));
 	public decimal MaxCoinjoinMiningFeeRate => GetEffectiveValue<DecimalValue, decimal>(nameof(MaxCoinjoinMiningFeeRate));
-	public int AbsoluteMinInputCount => int.Max(
-		GetEffectiveValue<IntValue, int>(nameof(AbsoluteMinInputCount)),
-		Constants.AbsoluteMinInputCount);
+
+	public int AbsoluteMinInputCount => Network == Network.Main ?
```

Comment:
This overrides whatever config the user specify. However that's not the problem, the problem is that the behavior has to be the same for all real networks. In other words, `Network == Network.Main` must be avoided as much as possible.

I would prefer to change the `AbsoluteMinInputCount` to `2` instead

Reply:
It's annoying to have to change config all the time you want to change something but OK I did it